### PR TITLE
mvcc: remove capnslog

### PR DESCRIPTION
**Pull request by jingyih**
Remove dependency of `capnslog` in mvcc pkg.

Related: #11426

### CODE REVIEW

**Comment by jingyih** [comment]
> @jingyih thanks! So as we get rid of capnslog, not under this PR but eventually I believe doc content like this https://github.com/etcd-io/etcd/blob/master/Documentation/dev-internal/logging.md is not relevant and should be updated.

Thanks for reminding! That doc needs to be updated with zap logger.

**Comment by spzala** [comment]
> > @jingyih thanks! So as we get rid of capnslog, not under this PR but eventually I believe doc content like this https://github.com/etcd-io/etcd/blob/master/Documentation/dev-internal/logging.md is not relevant and should be updated.
> 
> Thanks for reminding! That doc needs to be updated with zap logger.

Yrw and thanks too @jingyih Please let me know if you need help there!

File: mvcc/kvstore.go [hunk 1]
```diff
@@ -372,16 +369,12 @@ func (s *store) restore() error {
...
-			s.lg.Info(
-				"restored last compact revision",
-				zap.String("meta-bucket-name", string(metaBucketName)),
-				zap.String("meta-bucket-name-key", string(finishedCompactKeyName)),
-				zap.Int64("restored-compact-revision", s.compactMainRev),
-			)
-		} else {
-			plog.Printf("restore compact to %d", s.compactMainRev)
-		}
+		s.lg.Info(
```
Comment (spzala) [inline mvcc/kvstore.go]: @jingyih - actually, one comment - don't we need to set the log for the store`s.lg = zap.NewNop()` ?

Comment (jingyih) [reply mvcc/kvstore.go]: This is a good question. In etcd we use the store created by `newWatchableStore()`. I made sure the logger is not nil in that function, therefore s.lg will not be nil here. But I should apply your suggestion for extra safety.

Comment (jingyih) [reply mvcc/kvstore.go]: Fixed here: https://github.com/etcd-io/etcd/blob/6b389bf23ce298f6d33bb9a2b85e3873609dc440/mvcc/kvstore.go#L116

Comment (spzala) [reply mvcc/kvstore.go]: Awesome! Thanks @jingyih

**Pull request merged**